src/pages/Anix/main.ts

```diff
@@ -0,0 +1,127 @@
+import { Animeworld } from '../Animeworld/main';
+import { pageInterface } from '../pageInterface';
+import { SyncPage } from '../syncPage';
+
+export const Anix: pageInterface = {
+  domain: 'https://anix.to',
+  languages: ['English'],
+  name: 'Anix',
+  type: 'anime',
+  isSyncPage(url: string): boolean {
+    return true;
+  },
+  sync: {
+    getTitle(url: string): string {
+      return j.$('h1.anime-name').text().trim();
```

Comment:
Does not work

Reply:
Can you add details, because it does work but just not every time, I believe ?

Reply:
Now it works with `database: '9anime'` it looks like.

Reply:
Fixed, change to `.ani-name` instead of `anime-name`.